astro/src/content/docs/lifecycle/register-users/advanced-registration-forms.mdx

```diff
@@ -488,7 +489,7 @@ Advanced user registration forms add custom data to your users' profiles. Howeve
 
 There are two types of profile data:
 
-* User data, which is associated with the user. This could be in standard fields such as `mobilePhone` or custom data fields in `user.data`.
+* User data, which is associated with the user. This could be in standard fields such as `user.phoneNumber` or custom data fields in `user.data`.
```

Comment:
Is there another standard field we could use as an example besides an identity field?

Reply:
Yep. Picked birthDate